src/System.Numerics.Vectors/tests/Matrix3x2Tests.cs

```diff
@@ -1009,7 +1009,7 @@ struct Matrix3x2PlusFloat_2x
 
         // A test to make sure the fields are laid out how we expect
         [Fact]
-        [ActiveIssue(1002)]
+        [ActiveIssue("NullReferenceException")]
```

Comment:
That was a result of some earlier work done internally. These should remain ActiveIssue(1002), @chcosta could you please make these changes internally and then close this PR so that the merge will happen again.